backend/entities/AccessRequest.cs

```diff
@@ -44,14 +44,14 @@ public class AccessRequest : BaseEntity
         /// get - the list of agencies that the user is requesting to be added to.
         /// </summary>
         /// <returns></returns>
-        public ICollection<AccessRequestAgency> Agencies { get; } = new List<AccessRequestAgency>();
+        public ICollection<AccessRequestAgency> Agencies { get; private set; } = new List<AccessRequestAgency>();
```

Comment:
Not sure the value of a `private set` for EF entities.

Reply:
Profile wasn't working without this.